plugins/modules/database/misc/redis_incr.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Andreas Botzner <andreas at botzner dot com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: redis_incr
+short_description: Increment keys in Redis
+description:
+   - Increment integer keys in Redis database by 1 and get new value.
+author: "Andreas Botzner (@paginabianca)"
+options:
+  key:
+    description:
+      - Database key.
+    type: str
+    required: True
+  increment:
+    description:
+      - Amount to increment by. Can be integer of float.
+    required: False
+    type: str
```

Comment:
```suggestion
    type: float
```
Might be better to have two (mutually exclusive) options: `int_increment` and `float_increment`. That avoids having to use `type: float`, which might destroy some large integer values when they cannot be exactly represented by floats.

Reply:
mutually exclusive options sounds good :ok_hand: